isomorphe/batch.py

```diff
@@ -1,9 +1,58 @@
 from dataclasses import dataclass
 from enum import IntEnum, StrEnum
+from typing import Iterator
+
+from lxml.etree import _ListErrorLog as ETListErrorLog
 
 from isomorphe.geonetwork import MefArchive, MetadataType, WorkflowState
 
 
+@dataclass(kw_only=True)
+class TransformError:
+    message: str
+    line: int
+    column: int
+    domain_name: str
+    domain: int
+    type_name: str
+    type: int
+    level_name: str
+    level: int
+    filename: str
```

Comment:
From https://lxml.de/xpathxslt.html#errors-and-messages

We might not need everything, but at least we don't have to decide...